components/notion/actions/create-page-from-database/create-page-from-database.mjs

```diff
@@ -69,6 +70,10 @@ export default {
     },
   },
   async run({ $ }) {
+    if (this?.Tags && typeof this?.Tags === "string") {
```

Comment:
@lcaresia the reason for `buildPageProperties()` is so that this kind of conversion doesn't have to be copy and pasted in every action. Please look at [this common file here](https://github.com/PipedreamHQ/pipedream/blob/master/components/notion/common/notion-page-properties.mjs#L52).